src/features/transfer/useTokenTransfer.ts

```diff
@@ -83,16 +84,20 @@ export function useTokenTransfer(onDone?: () => void) {
         const hypErc20 = isNativeToRemote
           ? getHypErc20CollateralContract(tokenRoute.hypCollateralAddress, provider)
           : getHypErc20Contract(tokenRoute.sourceTokenAddress, provider);
+        const gasPayment = await hypErc20.quoteGasPayment(destinationChainId);
         const transferTxRequest = await hypErc20.populateTransaction.transferRemote(
           destinationChainId,
           utils.addressToBytes32(recipientAddress),
           weiAmount,
+          {
+            value: gasPayment,
+          },
```

Comment:
should the SDK generally be returning results from `populateTransaction`?

Reply:
Would be good yeah. Having both options (`prepareTransaction` and `sendTransaction`) would be ideal